### ISSUE TITLE
Scrolling slow on Mac

### ISSUE THREAD

**Issue by gwww** [wez/wezterm#692]
## Describe the bug

Pressing in holding the `j` key in `vim` or `neovim` scrolls very slowly compared to `iTerm2`

## Environment (please complete the following information):

 - OS: MacOS 11.1 (MacBook Pro 13")
 - Version: 20210405-110924-a5bb5be8-70-g2bb0e4e7
 - The active keyboard layout name: ENG

## To Reproduce

In `vim` edit a file (I used a ~120 line file). Press and hold `j` key. Scrolling is slow. Do the same in `iTerm2` scrolling is ~5x faster.

## Configuration

I don't believe anything relevant, but here is my config: https://github.com/gwww/dotfiles/tree/master/.config/wezterm

## Expected behavior
Faster scrolling.

## Other

I have a `wt-record` log running the second scenario. Github won't let me attach a `.tgz` file. Here's a dropbox link: https://www.dropbox.com/s/a8qwx9y5pjp4zl6/20210413132015.tgz?dl=0

I also have two screen recordings. Both running `vim` and holding down the `j` key to scroll through a file. On `iTerm2` it scrolls through about 120 lines in ~2 seconds. On `wezterm` its more like 12 seconds.

https://user-images.githubusercontent.com/3060128/114565338-2b676080-9c3f-11eb-94f8-d23dd7fd0ef5.mov

https://user-images.githubusercontent.com/3060128/114565356-2efae780-9c3f-11eb-992d-b88495469263.mov

Startup of `wezterm`:
```
 2021-04-13T13:20:15.815Z INFO  wezterm_mux_server_impl::local > setting up /Users/glenn/.local/share/wezterm/gui-sock-71746
 2021-04-13T13:20:16.007Z INFO  wezterm_gui::termwindow        > OpenGL initialized! Intel(R) Iris(TM) Plus Graphics 650 4.1 INTEL-16.1.7 is_context_loss_possible=false wezterm version: 20210405-110924-a5bb5be8-70-g2bb0e4e7
```

**Comment by bew** [comment]
_joke mode on_
This is a feature! So you're forced to learn the proper vim motions to navigate code, instead of spamming j/k to move around!
_joke mode off_
:grin:

**Comment by gwww** [comment]
Additional info (just so I don't get tagged vim guy 😉) when I try `fzf` scrolling up/down there is no apparent performance difference between `iTerm2` and `wezterm`. Don't know if that helps. Only thing that comes to my mind is maybe something to do with the way screen redraws or cursor movements work in `vim`.

**Comment by wez** [comment]
On my M1 mac, both wezterm and iterm have the same speed.

Someone else reported slow performance in vim and traced it to a vim plugin (airline, I think).
We then figured out that it was slow due to some unexpected font resolution overhead from the various symbol fonts and things that were in use.

I'd suggest running:

`WEZTERM_LOG=trace wezterm 2> /tmp/wezterm.txt`

and repeating that experiment and attaching that trace.  That should help us understand a bit more about where the time is going.

**Comment by gwww** [comment]
I can confirm when I turn off airline that the speed is comparable to iTerm2. Trace file, using vim, airline turned on (on DropBox as too big to attach here): https://www.dropbox.com/s/3pz69i0678lxznz/wezterm.txt?dl=0

**Comment by wez** [comment]
Please update to a more recent nightly; you're running 2bb0e4e797fb9fa613c0c2fbbfec1ecd5c4d1d90
which has fallback font performance issues that were resolved in 22c4407ae94a80347c93571aa554f0f603be9eb7

**Comment by gwww** [comment]
Yup! Thanks!
